### ISSUE TITLE
Properties cannot be named `id`

### ISSUE THREAD

**Issue by rosseljost** [jgraph/drawio#3460]
### Preflight Checklist

* [x] I agree to follow the [Code of Conduct](https://github.com/jgraph/drawio/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for a feature request that matches the one I want to file, without success.

**Describe the bug**
When adding a data property to an element, one cannot name it `id` otherwise it will not be stored.

**To Reproduce**
1. Create an element and give it a property with the name `id` and any value. E.g.:
    ![image](https://user-images.githubusercontent.com/44645301/227491794-f07455de-93d4-4ccd-91db-397d567a5f4b.png)
3. Export to SVG works with the `svgdata` plugin (that's why I didn't notice at first):
```svg
<g id="cell-MN4Gi1lTzegU4kbtGvQS-1"
            content="&lt;object label=&quot;&quot; test=&quot;test&quot; id=&quot;test-id&quot;/&gt;"
            data-label="" data-test="test" data-id="test-id">
     <rect x="0" y="0" width="120" height="60" fill="rgb(255, 255, 255)"
                stroke="rgb(0, 0, 0)" pointer-events="all" />
</g>
```
4. Saving the file will result in a loss of the information:
```svg
<object label="" test="test" id="MN4Gi1lTzegU4kbtGvQS-1">
	<mxCell style="rounded=0;whiteSpace=wrap;html=1;comic=0;" vertex="1" parent="1">
		<mxGeometry x="620" y="390" width="120" height="60" as="geometry"/>
	</mxCell>
</object>
```

**Expected behavior**
An error/warning that you can't use `id` as an attribute (and any other already used names for that matter).

**draw.io version (In the Help->About menu of the draw.io editor):**

- draw.io version 20.8.16

**Desktop (please complete the following information):**
 - OS: (Arch) Linux

**Comment by alderg** [comment]
Thanks for the report. This will be fixed in the next release.

**Comment by davidjgraph** [comment]
Released in 21.1.2
